libobs/media-io/audio-io.c

```diff
@@ -190,15 +190,18 @@ static void input_and_output(struct audio_output *audio, uint64_t audio_time,
 	/* get new audio data */
 	success = audio->input_cb(audio->input_param, prev_time, audio_time,
 				  &new_ts, active_mixes, data);
-	if (!success)
-		return;
+	if (!success) {
+		return false;
+	}
```

Comment:
I don't think adding `{` and `}` is necessary.
```suggestion
	if (!success)
		return false;
```